change/@fluentui-react-input-7c91bd48-6890-462b-ad49-2a529ab1bfed.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Can you merge with master (which already have the change to react-input) and remove this change file?